src/docs/product/alerts-notifications/index.mdx

```diff
@@ -8,19 +8,40 @@ description: "Sentry categorizes notifications into alerts and other types of no
 
 Sentry categorizes notifications broadly into alerts and non-alert notifications.
```

Comment:
Can we give a little more info here about the impact of these alerts and notifications? Because all this tells the user is what Sentry does (useful), but we want to explain better what the impact is of their choices here, right? We get a lot of comments that Sentry is too noisy, so here's where users can understand why and how to set things up so communications are useful and valid for their org.